orchestra/accounts/forms.py

```diff
@@ -28,7 +30,26 @@ class WorkerForm(forms.ModelForm):
     class Meta:
         model = Worker
         fields = ('slack_username', 'phone')
-        exclude = ('user', 'start_datetime')
+        exclude = ('user', 'start_datetime', 'slack_id')
+
+    def clean_slack_username(self):
+        slack_username = self.cleaned_data.get('slack_username')
+        if slack_username:
+            slack = Slacker(settings.SLACK_EXPERTS_API_KEY)
```

Comment:
i feel like slack implementation stuff should live in a `slack` module that we call, without having to pass in an api key, etc.